content/articles/creating-a-hand-gesture-volume-controller-using-python-and-pycharm/index.md

```diff
@@ -0,0 +1,261 @@
+
+### Referral hand image
+![hand](/engineering-education/creating-a-hand-gesture-volume-controller-using-python-and-pycharm/hand.jpg)
+
+
+*[Image Source: Aniketiq](https://www.bing.com/images/search?view=detailV2&ccid=A16aeeUy&id=2B00C1DE13C6EFD132E8FCABF5CF0A11695E1FCA&thid=OIP.A16aeeUyQVRHqGt_rphzJQHaCl&mediaurl=https%3a%2f%2fcdn.hashnode.com%2fres%2fhashnode%2fimage%2fupload%2fv1629398588657%2fboG68w85n.png%3fauto%3dcompress%2cformat%26format%3dwebp&cdnurl=https%3a%2f%2fth.bing.com%2fth%2fid%2fR.035e9a79e532415447a86b7fae987325%3frik%3dyh9eaREKz%252fWr%252fA%26pid%3dImgRaw%26r%3d0&exph=538&expw=1543&q=mediapipe+hand+tracking+points&simid=607990468186358942&FORM=IRPRST&ck=C941CC346294E5FC99172E9ED31C9E5B&selectedIndex=4)*
+
+The above image shows the numbers of the points that mediapipe uses to refer to different points of the hand. In this tutorial, we are going to use point 4 and point 8 which are the thumb and the index finger respectively.
+
+### Creating a hand gesture volume controller
+First of all, we will prepare our workspace. Launch the pycharm app. Click on the create a new project. This is shown in the screenshot provided below.
+
+![Pycharm](/engineering-education/creating-a-hand-gesture-volume-controller-using-python-and-pycharm/pycharm.png)
+
+Click on the create button on the window that appears next.
+
+We now need to install the libraries we need to use in our project. To do so, follow the following steps. To install numpy, open the terminal and type the following command. 
+```bash
+pip install numpy
+```
+*numpy* will help us work with arrays.
+
+![Terminal](/engineering-education/creating-a-hand-gesture-volume-controller-using-python-and-pycharm/terminal.png)
+
+The above screenshot shows where to type the command for numpy installation.
+
+Wait for a few seconds till numpy is installed successfully. We are going to repeat the same steps for the other libraries. We will only replace the command part. Use the following commands for the respective libraries.
+```bash
+ pip install opencv-python
+
+```
+ This library is imported as *cv2*. We will use this library to capture an image using the webcam and convert the image to RGB.
+ ```bash
+pip install mediapipe
+
+ ```
+ This is an open-source library developed by Google. It is used for both face and gesture recognition. For the matter of this tutorial, we will use it for hand gesture recognition.
+ ```bash
+pip install pycaw
+ ```
+  We will need this library to access the device's speaker and its master volume.
+ ```bash
+pip install python-math
+ ```
+ We will use this library to find the distance between point number 4 (the thumb) and point number 8 (the index finger) using hypotenuse.
+ ```bash
+pip install gpib-ctypes, comtypes
+ ```
+ These two libraries are depended upon by pycaw. Ctypes to provide C language compatible data types. Comtypes bases on the ctypes FFI(Foreign Function Interface) library.

```

Comment:
Consider using (``) when referring to libraries. For example, `pycaw`. It helps in highlighting them.